search/kv/kv.md

```diff
@@ -34,6 +29,14 @@ x:1|y:2|z:3|foo:bar
 * `-d <delimiters>`: The "-d" flag specifies the delimiters to use. You can specify multiple delimiters characters; for example, to use double-quote, tab, and space as delimiters, set `-d "\" \t"`.
 * `-s`: The "-s" option puts the module into strict mode. In strict mode, an entry will be dropped unless *all* specified extractions succeed.
 * `-q`: The "-q" option enables quoted values. This allows values to contain the delimiter characters, e.g. `key="this is the value"`
+* `-ld`: The "-ld" option enables handling strangely delimited values such as ``.  See the note below.
```

Comment:
Should there be something in the block quotes here?

Reply:
@djfritz PTAL.  good catch